integrations/docker/build.sh

```diff
@@ -59,7 +59,12 @@ set -ex
 
 [[ -n $VERSION ]] || die "version cannot be empty"
 
-mb_space_before=$(df -m /var/lib/docker/ | awk 'FNR==2{print $3}')
+#Fix storage location of Docker images and containers for osx or linux
```

Comment:
Could we update the comment? I am unsure how `Fix the storage location...` helps me read the code. I read the code as gdarwin and non-darwin have different commands to compute docker space left`

Lets drop the entire comment if nothing can be added to enhance code readability. The most I could think of would be that these rely on default paths for docker (can one set non-default) but that also seems obvious reading the code.

Reply:
Thank you for your response. Of course you are right, the comment is redundant. Thanks